pyhap/characteristic.py

```diff
@@ -184,12 +200,11 @@ def set_value(self, value, should_notify=True):
     def client_update_value(self, value):
         """Called from broker for value change in Home app.
 
-        Change self.value to value and call callback.
+        Call set_value and call callback.
         """
-        logger.debug('client_update_value: %s to %s',
-                      self.display_name, value)
         self.value = value
         self.notify()
+        # Call setter_callback
```

Comment:
Can you please revert the changes to `client_update_value`? I don't think they are necessary for `get_value`.